lib/ain-ocean/Cargo.toml

```diff
@@ -0,0 +1,23 @@
+[package]
+name = "ain-ocean"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+tokio = { version = "1", features = ["full"] }
+serde = { version = "1.0", features = ["derive"] }
+tokio-serde = { version = "0.8.0", features = ["json"] }
+log = "0.4"
+futures = "0.3.29"
+async-trait = "0.1.74"
+anyhow = "1.0.75"
+jsonrpsee =  { version = "0.20.3", features = ["server", "macros", "http-client", "jsonrpsee-server"] }
```

Comment:
jsonrpcsee version - let's keep the whole codebase in a single version and upgrade as a set. 

We'll need to upgrade our existing one up. But until then, use the previous.